scripts/jenkins-yoctobuild-build.sh

```diff
@@ -177,5 +185,32 @@ if [ "$RUN_INTEGRATION_TESTS" = "true" ]; then
     if [ $PUBLISH_ARTIFACTS = "true" ]; then
         s3cmd -F put core-image-full-cmdline-vexpress-qemu.ext4 s3://mender/temp/core-image-full-cmdline-vexpress-qemu.ext4
         s3cmd setacl s3://mender/temp/core-image-full-cmdline-vexpress-qemu.ext4 --acl-public
+
+        if [ ! -z ${RELEASE_VERSION} ]; then
+
+            cd $BUILDDIR/tmp/deploy/images/vexpress-qemu/
```

Comment:
I thought the reason for the `cp` you added above was to do `cd $WORKSPACE/vexpress-qemu` instead of this random dir. Or did I misunderstand?